.github/workflows/main.yml

```diff
@@ -17,11 +17,89 @@ jobs:
+          - {ROS_DISTRO: kinetic, UPSTREAM_WORKSPACE: file, ROSINSTALL_FILENAME: .i.do.not.exist, EXPECT_EXIT_CODE: 1}
+          - {DOCKER_IMAGE: 'ros:kinetic-ros-base', ROS_REPO: ros, NOT_TEST_BUILD: true, DEBUG_BASH: true, VERBOSE_OUTPUT: false, DOCKER_COMMIT: img_temp, POST_PROCESS: 'eval docker image inspect $DOCKER_COMMIT --format="$DOCKER_COMMIT:\ \"{{.Size}}\" bytes"'}
+          - {ROS_DISTRO: lunar, ROS_REPO: 'ros-shadow-fixed', TARGET_WORKSPACE: 'industrial_ci/mockups/industrial_ci_testpkg'}
+          - {ROS_DISTRO: melodic, AFTER_SCRIPT: 'grep -q ID=ubuntu /etc/os-release && grep -q VERSION_CODENAME=bionic /etc/os-release'}
+          - {ROS_DISTRO: melodic, BEFORE_INIT: 'grep -q ID=debian /etc/os-release && grep -q VERSION_ID=\"9\" /etc/os-release', EXPECT_EXIT_CODE: 1}
+          - {ROS_DISTRO: melodic, OS_NAME: debian, OS_CODE_NAME: stretch, AFTER_SCRIPT: 'grep -q ID=debian /etc/os-release && grep -q VERSION_ID=\"9\" /etc/os-release'}
+          - {ROS_DISTRO: melodic, OS_NAME: debian, EXPECT_EXIT_CODE: 1}
+          - {ROS_DISTRO: melodic, OS_NAME: debian, OS_CODE_NAME: bionic, EXPECT_EXIT_CODE: 1}
+          - {ROS_DISTRO: melodic, ROS_REPO: ros, BUILDER: colcon, AFTER_SCRIPT: 'rosenv && [ "$CMAKE_PREFIX_PATH" = "/root/target_ws/install/industrial_ci:/opt/ros/melodic" ]'}
+          - {ROS_DISTRO: noetic, TARGET_WORKSPACE: 'industrial_ci/mockups/industrial_ci_testpkg'}
+          - {ROS_DISTRO: noetic, PRERELEASE: true}
+          - {ROS_DISTRO: foxy}
+          - {ROS_DISTRO: rolling}
+
           - {ROS_DISTRO: melodic}
           - {ROS_DISTRO: melodic, ROS_REPO: main}
+
+          # Format tests
+          - {ROS_DISTRO: indigo, TARGET_WORKSPACE: 'industrial_ci/mockups/format_tests/cpp/LLVM', CLANG_FORMAT_CHECK: 'LLVM', CLANG_FORMAT_VERSION: 3.8}
+          - {ROS_DISTRO: kinetic, TARGET_WORKSPACE: 'industrial_ci/mockups/format_tests/cpp/LLVM', CLANG_FORMAT_CHECK: 'LLVM'}
+          - {ROS_DISTRO: kinetic, TARGET_WORKSPACE: 'industrial_ci/mockups/format_tests/cpp/WebKit', CLANG_FORMAT_CHECK: 'LLVM', EXPECT_EXIT_CODE: 1}
+          - {ROS_DISTRO: kinetic, TARGET_WORKSPACE: 'industrial_ci/mockups/format_tests/cpp/WebKit', CLANG_FORMAT_CHECK: 'file'}
+          - {ROS_DISTRO: kinetic, TARGET_WORKSPACE: 'industrial_ci/mockups/format_tests/cpp/LLVM', CLANG_FORMAT_CHECK: 'WebKit', EXPECT_EXIT_CODE: 1}
+
+          # Tidy
+          - {ROS_DISTRO: melodic, TARGET_WORKSPACE: 'industrial_ci/mockups/test_clang_tidy', CLANG_TIDY: pedantic}
+          - {ROS_DISTRO: melodic, TARGET_WORKSPACE: 'industrial_ci/mockups/test_clang_tidy', CLANG_TIDY: pedantic, CLANG_TIDY_ARGS: "-checks=modernize-use-nullptr", EXPECT_EXIT_CODE: 1}
+        
+        include:
+          - env: {ROS_DISTRO: kinetic, NOT_TEST_BUILD: true, NOT_TEST_INSTALL: true, POST_PROCESS: 'I_am_supposed_to_fail'}
+            experimental: true
```

Comment:
Is this still needed? This job doesn't fail any more.

Reply:
This was only needed to verify that Travis did not jump out before reaching the end of the script..
But for reason I have removed the [line](https://github.com/ros-industrial/industrial_ci/blame/c371892602e0e0af3f4abd5cf4b0e11d83badbc9/.travis.yml#L74), which would make it fail.
Just remove it :)